realm/realm-library/src/main/java/io/realm/sync/permissions/PermissionUser.java

```diff
@@ -72,4 +74,18 @@ public String getId() {
     public @Nullable  RealmResults<Role> getRoles() {
         return roles;
     }
+
+    /**
+     * The user's private role. This will be initialized to a role named for the user's
```

Comment:
I think this description is slightly misleading since the user can set it to anything after the fact. This actually works more like `tag` in a ViewHolder. I pushed for the semantics described here, but currently, that is not what is implemented.

Perhaps something like this instead.

```
Returns the role set by {@link #setRole(Role)}. This will be initialized to a role named for the user's identity that contains the user as its only member.
```

Reply:
We'll keep the original description since the `setRole` is removed now